db/vita_partners.yml

```diff
@@ -73,6 +65,9 @@ vita_partners:
   - fc
   states:
   - TX
+  - TN
+  - AR
+  - MS
```

Comment:
I routed these states to Foundation Communities, but is there a better way to share them across the "national pool"?

Reply:
I think it might be slightly preferable to keep the `states` list here to be only the states we've explicitly promised a partner, rather than making sure to list all of the states. Any missing states will be routed via the `accepts_overflow: true` option, which will go to FC anyway.

Reply:
Agreed. I made those changes, but I cannot merge, so I think you'll need to.